netlify/src/main/scala/feral/netlify/NetlifyHttpEvent.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package feral.netlify
+import io.circe.Decoder
+import cats.data.NonEmptyList
+
+final case class NetlifyHttpEvent(
```

Comment:
accurate based on typescript model in https://github.com/netlify/functions/blob/86697792ef363211f367cbf2fab5334389a739ed/src/function/event.ts